### ISSUE TITLE
Sharing a sync from desktop to mobile fails

### ISSUE THREAD

**Issue by vendion** [syncthing/syncthing-android#510]
--- Desktop ---
OS: Arch Linux
SyncThing: v 0.12.7

-- Android --
OS: 6.0.0
SyncThing: 0.12.6

When shareing the default "Sync" directory with my phone from my computer, but on my phone I'm never prompted if I want to add the share.

_SyncThing logs_
```
--------- beginning of main
12-07 20:13:44.202 14700 14726 I SyncthingNativeCode: [NF7TJ] 01:13:44 INFO: New UPnP port mapping: external port 10664 to local port 22000.
12-07 20:13:57.053 14700 14726 I SyncthingNativeCode: [NF7TJ] 01:13:57 INFO: Established secure connection to 44RIFIE-VSJK6FW-MSAO7TF-MQ7XXGV-DDW43K5-TWZSF5T-FS4K32L-5HEOKA7 at 192.168.0.107:39956-173.230.140.226:22000 (direct-dial)
12-07 20:13:57.056 14700 14726 I SyncthingNativeCode: [NF7TJ] 01:13:57 INFO: Unexpected folder ID "default" sent from device "44RIFIE-VSJK6FW-MSAO7TF-MQ7XXGV-DDW43K5-TWZSF5T-FS4K32L-5HEOKA7"; ensure that the folder exists and that this device is selected under "Share With" in the folder configuration.
12-07 20:13:57.059 14700 14726 I SyncthingNativeCode: [NF7TJ] 01:13:57 INFO: Device 44RIFIE-VSJK6FW-MSAO7TF-MQ7XXGV-DDW43K5-TWZSF5T-FS4K32L-5HEOKA7 client is "syncthing v0.12.7" named "Beyla"
12-07 20:13:57.208 14700 14726 I SyncthingNativeCode: [NF7TJ] 01:13:57 INFO: Connected to already connected device (44RIFIE-VSJK6FW-MSAO7TF-MQ7XXGV-DDW43K5-TWZSF5T-FS4K32L-5HEOKA7)
12-07 20:14:10.954 14700 14726 I SyncthingNativeCode: [NF7TJ] 01:14:10 INFO: Established secure connection to WRV6JXB-E6XNQNM-PWI2AQL-YMLR44C-UZTCSWO-G7Y5UFO-UZDQZMA-VRO4UQX at 192.168.0.107:42455-192.168.0.101:22000 (direct-dial)
12-07 20:14:10.958 14700 14726 I SyncthingNativeCode: [NF7TJ] 01:14:10 INFO: Device WRV6JXB-E6XNQNM-PWI2AQL-YMLR44C-UZTCSWO-G7Y5UFO-UZDQZMA-VRO4UQX client is "syncthing v0.12.6" named "Baldr"
12-07 20:14:10.959 14700 14726 I SyncthingNativeCode: [NF7TJ] 01:14:10 INFO: Unexpected folder ID "default" sent from device "WRV6JXB-E6XNQNM-PWI2AQL-YMLR44C-UZTCSWO-G7Y5UFO-UZDQZMA-VRO4UQX"; ensure that the folder exists and that this device is selected under "Share With" in the folder configuration.
12-07 20:14:10.962 14700 14726 I SyncthingNativeCode: [NF7TJ] 01:14:10 INFO: Unexpected folder ID "default" sent from device "WRV6JXB-E6XNQNM-PWI2AQL-YMLR44C-UZTCSWO-G7Y5UFO-UZDQZMA-VRO4UQX"; ensure that the folder exists and that this device is selected under "Share With" in the folder configuration.
```

-- System logs --
http://sprunge.us/cVXM

**Comment by licaon-kter** [comment]
duplicate of https://github.com/syncthing/syncthing-android/issues/295

**Comment by AudriusButkevicius** [comment]
On Android you have to add it manually, with the same name.

**Comment by vendion** [comment]
@licaon-kter Sorry about the duplicate, I tried searching through the issue queue, but didn't find one similar and honestly this didn't happen `The only way to be notified of a folder being shared from the desktop client with the Android device is to have the Web UI open at the time the folder is shared.` which is what made me think it was unrelated.  The only why I knew that my phone saw that the directory was being shared with it was through the logs.

@AudriusButkevicius Thanks, I manually added it an it and I see my files!

It would be nice if manually adding the sync wasn't necessary on Android.
